spec/examples.txt

```diff
@@ -0,0 +1,20 @@
+example_id                                      | status | run_time        |
+----------------------------------------------- | ------ | --------------- |
+./spec/views/posts/index.html.erb_spec.rb[1:1]  | passed | 0.38823 seconds |
+./spec/views/posts/index.html.erb_spec.rb[1:2]  | passed | 0.05713 seconds |
+./spec/views/posts/index.html.erb_spec.rb[1:3]  | passed | 0.05351 seconds |
+./spec/views/posts/index.html.erb_spec.rb[1:4]  | passed | 0.07366 seconds |
+./spec/views/posts/index.html.erb_spec.rb[1:5]  | passed | 0.05658 seconds |
+./spec/views/posts/index.html.erb_spec.rb[1:6]  | passed | 0.05701 seconds |
+./spec/views/posts/show.html.erb_spec.rb[1:1:1] | failed | 0.04863 seconds |
+./spec/views/posts/show.html.erb_spec.rb[1:1:2] | failed | 0.05195 seconds |
+./spec/views/posts/show.html.erb_spec.rb[1:1:3] | failed | 0.4734 seconds  |
+./spec/views/posts/show.html.erb_spec.rb[1:1:4] | failed | 0.0486 seconds  |
+./spec/views/posts/show.html.erb_spec.rb[1:1:5] | failed | 0.05526 seconds |
+./spec/views/posts/show.html.erb_spec.rb[1:1:6] | failed | 0.04629 seconds |
+./spec/views/users/index.html.erb_spec.rb[1:1]  | passed | 0.03077 seconds |
+./spec/views/users/index.html.erb_spec.rb[1:2]  | passed | 0.3544 seconds  |
+./spec/views/users/show.html.erb_spec.rb[1:1]   | passed | 0.05793 seconds |
+./spec/views/users/show.html.erb_spec.rb[1:2]   | passed | 0.41124 seconds |
+./spec/views/users/show.html.erb_spec.rb[1:3]   | passed | 0.05489 seconds |
+./spec/views/users/show.html.erb_spec.rb[1:4]   | passed | 0.07655 seconds |
```

Comment:
As I mentioned above all your tests to pass(`rspec spec/views`), you can even see from your `examples.txt` file the list of all the failed tests shown in the picture below:
![Screenshot from 2022-11-02 02-02-04](https://user-images.githubusercontent.com/83042742/199365227-e8b5bb81-8db9-4137-ac54-6c4f4e795366.png)